marin/resources.py

```diff
@@ -96,6 +151,40 @@ def as_remote_kwargs(self) -> dict:
     def as_ray_resources(self) -> RayResources:
         return RayResources(**self.as_remote_kwargs())
 
+    def device_flops(self) -> float:
+        """Get the peak FLOPs/s for the GPU type."""
+        if self.device_flops_override is not None:
+            logger.info(f"Using user-provided device FLOPS override: {self.device_flops_override} for GPU type {self.accelerator_type}")
+            return self.device_flops_override
+
+        from marin.resources_utils import device_flops_map
+        
+        if self.accelerator_type is None:
+            raise ValueError(
+                "accelerator_type must be explicitly specified in GpuConfig. "
+                "Available types: " + ", ".join(_ACCEL_TYPES)
```

Comment:
maybe add a note that if you want to use an accelerator not supported by ray, you can specify device_flops_override

Reply:
also you can't hit this right?